pom.xml

```diff
@@ -564,7 +564,7 @@
                     <connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
                     <scmVersionType>tag</scmVersionType>
                     <scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
-                    <goals>install -DskipTests</goals>
+                    <goals>install -DskipTests -Dmaven.site.skip=true</goals>
```

Comment:
I would extract this modification to a separate task just because it's a bit OOS for this task and skipping site should be done also for other eCommerce services that build eCommerce local library to speed up overall building time with other interventions, such as use cache from prev builds